### ISSUE TITLE
small/collapsed edit toolbar usability: Don't change button contents on hover

### ISSUE THREAD

**Issue by fredvd** [plone/Products.CMFPlone#1610]
After doing Plone 4 stuff for a few months and coming back to a Plone 5 site, what struck me immediately upon logging in is getting mentally nervous from the vertical Plone 5 edit toolbar. 

When I want to click on a button, or hover with my mouse vertically, as soon as I see the 'view' icon and want to click on it, the icon is gone. Apart from the slide in from the bottom that's adding to the nervousness, I think this a variation on one of the UX principles to not change/reorder navigation menu's on the fly by context (as Microsoft did in Office for a few versions by).  Adding a slight hover effect is fine as long as the previous image or text is still visible.

What I guess is also happening on a cognitive level is that I see an icon (picture, visual) and when I try to click on it there's suddenly a text (symbolic, processing) and after the initial split second doubt (am I still on the menu item I hovered to) this may also add to the confusion.

Consider this: wouldn't it be weird if we had only text in the toolbar buttons, but would show synonyms as soon as you hovered over them? "view -> display", "edit -> change", etc? 

If we did this for beginning users who cannot remember the iconic meaning of the toolbar items: they can expand the toolbar and see the explanations. The only benefit I see so far is that the clock item shows the modification date, cool from a screen real estate perspective, but nothing else.

**Comment by hvelarde** [comment]
also related with #1001

**Comment by jensens** [comment]
Closing as outdated (9 years old). About the old Plone 5 collapsed vertical toolbar hover behavior. Toolbar replaced in Plone 6.
If you think this issue is still relevant please reopen it. Thank you for the original report!
